terraform/Dockerfile

```diff
@@ -3,11 +3,17 @@ FROM alpine:3.6
 
 RUN apk add --no-cache unzip curl
 
+# install airworthy
+COPY airworthy /usr/local/bin/airworthy
```

Comment:
Airworthy should be download from github releases page and sha256sum check performed on it

See here: https://github.com/jetstack/tarmak/blob/master/terraform/amazon/kubernetes/templates/puppet_agent_user_data.yaml#L15